src/handlers/track_table.rs

```diff
@@ -105,7 +105,19 @@ pub fn handler(key: Key, app: &mut App) {
         Key::Ctrl('d') => {
             match &app.track_table.context {
                 Some(context) => match context {
-                    TrackTableContext::MyPlaylists => {}
+                    TrackTableContext::MyPlaylists => {
+                        if let (Some(playlists), Some(selected_playlist_index)) =
+                            (&app.playlists, &app.selected_playlist_index)
+                        {
+                            if let Some(selected_playlist) =
+                                playlists.items.get(selected_playlist_index.to_owned())
+                            {
+                                app.playlist_offset += 20;
```

Comment:
Instead of hardcoding `20`, please use `self.large_search_limit`

Reply:
I see, of course - I have fixed it to use the `large_search_limit` from app.